databricks-dbutils-scala/src/main/scala/com/databricks/sdk/scala/dbutils/SdkDBUtilsImpl.scala

```diff
@@ -152,3 +154,52 @@ private class SdkSecretsUtils(client: WorkspaceClient) extends SecretUtils with
       SecretScope(scope.getName)
     }
 }
+
+private class SdkJobsUtils extends JobsUtils with NoHelp {
+  override val taskValues: TaskValuesUtils = new SdkTaskValues
+}
+
+private class SdkTaskValues extends TaskValuesUtils with NoHelp {
+  private var commandContext =
+    CommandContext(rootRunId = None, currentRunId = None, jobGroup = None, tags = Map.empty, extraContext = Map.empty)
+
+  /**
+   * Sets a task value on the current task run. This method is a no-op if used outside of the job context.
+   *
+   * @param key
+   *   the task value's key
+   * @param value
+   *   the value to be stored (must be JSON-serializable)
+   */
+  override def set(key: String, value: Any): Unit = throw new NotImplementedError("set is not supported in Scala.")
+
+  /**
+   * Returns the latest task value that belongs to the current job run.
+   *
+   * @param taskKey
+   *   the task key of the task value
+   * @param key
+   *   the key of the task value
+   * @param default
+   *   the value to return when called inside of a job context if the task value does not exist (must not be None)
+   * @param debugValue
+   *   the value to return when called outside of a job context (must not be None)
+   * @return
+   *   the task value (if it exists) when called inside of a job context
+   */
+  override def get(taskKey: String, key: String, default: Option[Any], debugValue: Option[Any]): Any =
+    throw new NotImplementedError("get is not supported in Scala.")
+
+  override def setJson(key: String, value: String): Unit =
+    throw new NotImplementedError("setJson is not supported for local development in Scala.")
+
+  override def getJson(taskKey: String, key: String): Seq[String] = Seq.empty
+
+  override def getContext(): CommandContext = {
+    commandContext
+  }
+
+  override def setContext(context: CommandContext): Unit = {
+    commandContext = context
+  }
```

Comment:
Why do we implement these in the SDK? CommandContext is purely a DBR concept so it should be provided by DBR?

Reply:
I don't know, but it is a part of public interface now, should I throw error?

Reply:
Throw error, that it is not supported outside of DBR, is it OK @gaborratky-db?